frontend/src/features/Address/utils.ts

```diff
@@ -15,7 +15,7 @@ export const IsAddressWithValidPrefix = (addr: string): boolean => {
 export const GetPrefix = (n: CardanoNetworkType): string => {
 	switch (n) {
 	case CardanoNetworkType.VectorTestNetNetwork:
```

Comment:
I am not sure if we need to handle these vector types, Maybe just make this function simpler:
```
if n == CardanoNetworkType.TestNetNetwork {
   return "addr_test"
}
return "addr"
```